src/music/player.py

```diff
@@ -50,416 +35,432 @@ def __init__(self, voice_channels, ffmpeg_path, messenger: Messenger, text_chann
+        return EmbedFactory.create_embed(type='player', **vars(self))
+
+    async def update_player_embed(self) -> None:
+        """Edits the player message (if registered) with a newly generated embed."""
+        if self.player_message:
+            await self.player_message.edit(embed = self.player_embed)
+    
+    def default_player_metadata(self) -> None:
+        """Sets the player metadata variables to default values.
+        Note: Since footer is part of player, it will be reset as well."""
+
+        # Player metadata
+        self.display_description: str = False
+        self.video_title: str = None
+        self.video_url: str = None
+        self.video_description: str = None
+        self.video_uploader: str = None
+        self.video_uploader_url: str = None
+        self.video_thumbnail: str = None
+
+        # Footer metadata
+        self.icon_url: str = None
+        self.resolved = False
+        self.from_file = False
+
+        self.paused: bool = None
+        self.video_source: str = None
+        self.playhead: timedelta = None
+        self.duration: timedelta = None
+        self.sponsorblock: bool = None
+    
+    ########## Reaction Functions ##########
+    async def register_player_reactions(self, message: Message):
+        """Register all the reactions in this class"""
+        if self.player_reactions:
+            for reactions in self.player_reactions:
+                await reactions.register(message)
+
+    async def remove_player_reactions(self):
+        """Removes all the reactions in this class"""
+        if self.player_reactions:
+            for reactions in self.player_reactions:
+                await reactions.remove_all()
+
+    async def toggle_display_description(self):
+        """Intake function for 💬 emoji reaction on_press event"""
+        self.display_description = not self.display_description
+        await self.update_player_embed()
+
+    async def on_play_pause(self):
+        """Intake function for ⏯ emoji reaction on_press event"""
+        if self.connected_client:
+            if self.connected_client.is_playing():
+                self.pause()
+            elif self.connected_client.is_paused():
+                self.resume()
+            await self.update_player_embed()
+
+    ########## General Helper Functions ##########
+    async def register_command_channel(self, command_channel: TextChannel):
+        """Sends and registers all embeds that this player implements to the command channel"""
+
+        self.reporter_message = await command_channel.send(embed=self.reporter_embed)
+        self.lyrics_message   = await command_channel.send(embed=self.lyrics_embed)
+        self.queue_message    = await command_channel.send(embed=self.queue_embed)
+        self.player_message   = await command_channel.send(embed=self.player_embed)
+
+    def metadata_youtube_dl(self, info: dict) -> dict[str,str]:
```

Comment:
Move the functions below to helpers,py

Reply:
These types of changes are now in issue #73. It won't be done in this PR.